faqs.rst

```diff
@@ -20,3 +20,95 @@ How do I report a bug?
 ----------------------
 
 You can read up on reporting bugs here: :ref:`contributing` or report one directly for flux `core <https://github.com/flux-framework/flux-core/issues>`_ or `sched <https://github.com/flux-framework/flux-sched/issues>`_.
+
+.. _not_managing_all_resources:
+
+---------------------------------------------------------------------------------
+Why is Flux not discovering and managing all of the resources on the system/node?
+---------------------------------------------------------------------------------
+
+This can be due to various bind flags that need to be passed parallel launcher that started Flux. For example at LLNL you must pass ``--mpibind=off`` to ``srun`` and ``--bind=none`` to ``jsrun``.
+
+Also on all systems, Flux relies on hwloc to auto-detect the on-node resources
+available for scheduling.  The hwloc that Flux is linked against must be
+configured with ``--enable-cuda`` for Flux to be able to detect Nvidia GPUs.
+
+You can test to see if your system default hwloc is CUDA-enabled with:
+
+.. code-block:: sh
+
+  lstopo | grep CoProc
+
+If no output is produced, then your hwloc is not CUDA-enabled.
+
+.. _launch_large_num_jobs:
+
+------------------------------------------------------------
+How do I efficiently launch a large number of jobs to Flux?
+------------------------------------------------------------
+
+See `bulksubmission <https://github.com/flux-framework/flux-workflow-examples/tree/master/async-bulk-job-submit>`_ for an example workflow or flux `tree <https://github.com/flux-framework/flux-sched/blob/master/src/cmd/flux-tree>`_.
+
+.. _node_memory_exhaustion:
+
+------------------------------------------------------------------
+Memory exhaustion on a node when running large ensembles with Flux
+------------------------------------------------------------------
+
+Set ``rundir`` so that the content store is not saved to ``/tmp`` but to filesystem.
```

Comment:
We should add a sentence of context.  Sorry again for the shorthand notes in the FAQ issue:

```suggestion
Flux's in-memory KVS is backed by an on-disk content store.  The default location for the content store is ``/tmp``, which for some systems is configured as a RAMDisk.  To minimize Flux's memory footprint at the cost of a slower content store, set ``rundir`` so that the content store is not saved to ``/tmp`` but to the filesystem.
```